src/js/utils/FormUtil.js

```diff
@@ -0,0 +1,89 @@
+function cloneDefinition(array) {
+  return array.map(function (object) {
+    if (Array.isArray(object)) {
+      return object.slice();
+    }
+
+    if (typeof object === 'object') {
+      return Object.assign({}, object);
+    }
+
+    return object;
+  });
+}
+
+function getProp(key) {
+  return key.split('[')[0];
+}
+
+function getPropIndex(key) {
+  let value = key.split('[');
+  return parseInt(value[1].split(']')[0])
+}
+
+function getPropKey(key) {
+  let value = key.split('[');
+  return value[1].split('.')[1];
+}
+
+function isNotMultipleProp(key) {
+  return !key.includes('[') || !key.includes(']');
+}
+
+const FormUtil = {
+  getMultipleFields: function (prop, count, definition) {
```

Comment:
This is more a create multiple fields definition than a get fields functions, isn't it?